webapp/src/main/java/com/box/l10n/mojito/service/branch/notification/BranchNotificationMessageSendersConfigurationProperties.java

```diff
@@ -277,4 +287,19 @@ public void setNoMoreStrings(String noMoreStrings) {
       }
     }
   }
+
+  static class GithubConfigurationProperties {
+
+    // TODO(jean) it migth make more to look up by "id" vs "owner" to have all implementation
+    // with the same logic, instead of having logical keys that are implementation specific
+    String owner;
```

Comment:
@maallen thoughts?

Reply:
(As the IDs in other cases are strings - does it matter what we put in the IDs in general for the values as long as they're unique?)

Reply:
I think that should be fine, presumably in that case we'll need to store the notifier id in the database for each relevant branch (and specify the notifier id rather than owner type when creating the branch itself). I don't see an issue here as long as we're consistent in the approach between what's used in the cli command and the server backend